src/Sylius/Component/Grid/Provider/ArrayGridProvider.php

```diff
@@ -30,20 +40,35 @@
      */
     public function __construct(ArrayToDefinitionConverterInterface $converter, array $gridConfigurations)
     {
-        foreach ($gridConfigurations as $code => $gridConfiguration) {
-            if (isset($gridConfiguration['extends'], $gridConfigurations[$gridConfiguration['extends']])) {
-                $gridConfiguration = $this->extend($gridConfiguration, $gridConfigurations[$gridConfiguration['extends']]);
+        $this->converter = $converter;
+        $this->gridConfigurations = $gridConfigurations;
+    }
+
+    private function convertGrids()
```

Comment:
Missing return types

Reply:
Btw. could you move it below public and protected methods?